chainer/training/extensions/evaluator.py

```diff
@@ -164,7 +164,8 @@ def evaluate(self):
 
         for batch in it:
             observation = {}
-            with reporter_module.report_scope(observation):
+            with reporter_module.report_scope(
+                    observation), function.no_backprop_mode():
```

Comment:
`no_backprop_mode` is already set [here](https://github.com/pfnet/chainer/pull/2356/files#diff-da93b8a5a0251d65b0d064c38515bfe2R170).